### DISCUSSION TITLE
Jingle support

### DISCUSSION THREAD

**Discussion by lmedjebeur** [xmppjs/xmpp.js#946] [Q&A]
Hello ,

I use [@xmpp/client](https://www.npmjs.com/package/@xmpp/client) , working like a charm , but now I want to use Webrtc and Jingle with Prosody , I looking for a plugin for Jingle, Any idea to use xmpp.js with Jingle ? 
( I found a plugin for stropheJs )
Thanks you

**Comment by Saitejakolan9** [comment]
Hello! How to integrate Strophe.js in react-native?
